### PR TITLE
feat: created routes for dashboard, login & register

### PR DESCRIPTION
**Pull request by Wishmastering**
Routes Created

### CODE REVIEW

File: app/dashboard/page.tsx
Comment (humexxx) [inline app/dashboard/page.tsx:1]: If there are not inner pages in a folder there is not need to have a folder and you can simply go with dashboard.tsx

File: app/dashboard/page.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Link from 'next/link'
+
+export default function Home() {
```
Comment (humexxx) [inline app/dashboard/page.tsx]: Eventhough it is a default export the name of the component should reflect the page intent

Comment (humexxx) [reply app/dashboard/page.tsx]: Commets added apply for all files

Comment (Wishmastering) [reply app/dashboard/page.tsx]: Done

**Pull request merged**